recipes/cpr/all/conanfile.py

```diff
@@ -91,28 +95,69 @@ def _patch_sources(self):
+            "CPR_FORCE_USE_SYSTEM_CURL": "USE_SYSTEM_CURL",
+            "CPR_BUILD_TESTS": "BUILD_CPR_TESTS",
+            "CPR_BUILD_TESTS_SSL": "BUILD_CPR_TESTS_SSL",
+            "CPR_GENERATE_COVERAGE": "GENERATE_COVERAGE",
+            "CPR_USE_SYSTEM_GTEST": "USE_SYSTEM_GTEST",
+            "CPR_FORCE_OPENSSL_BACKEND": "USE_OPENSSL",
+            "CPR_FORCE_WINSSL_BACKEND": "USE_WINSSL",
+            "CPR_GENERATE_COVERAGE": "GENERATE_COVERAGE",
+        }
+
+        if self._uses_old_cmake_options:
+            # Get the translated option if we can, or the original if one isn't defined.
+            return CPR_1_6_CMAKE_OPTIONS_TO_OLD.get(option, option)
+        else:
+            return option
+
     def _configure_cmake(self):
         if not self._cmake:
             self._cmake = CMake(self)
-            self._cmake.definitions["USE_SYSTEM_CURL"] = True
-            self._cmake.definitions["BUILD_CPR_TESTS"] = False
-            self._cmake.definitions["GENERATE_COVERAGE"] = False
-            self._cmake.definitions["USE_SYSTEM_GTEST"] = False
+            self._cmake.definitions[self._get_cmake_option("CPR_FORCE_USE_SYSTEM_CURL")] = True
+            self._cmake.definitions[self._get_cmake_option("CPR_BUILD_TESTS")] = False
+            self._cmake.definitions[self._get_cmake_option("CPR_GENERATE_COVERAGE")] = False
+            self._cmake.definitions[self._get_cmake_option("CPR_USE_SYSTEM_GTEST")] = False
             if self._supports_openssl:
                 self._cmake.definitions["CMAKE_USE_OPENSSL"] = self.options.get_safe("with_openssl", False)
             if self._supports_winssl: # The CMake options changed
                 # https://github.com/whoshuu/cpr/commit/18e1fc5c3fc0ffc07695f1d78897fb69e7474ea9#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR39-R40
-                self._cmake.definitions["USE_OPENSSL"] = self.options.get_safe("with_openssl", False)
-                self._cmake.definitions["USE_WINSSL"] = self.options.get_safe("with_winssl", False)
+                self._cmake.definitions[self._get_cmake_option("CPR_FORCE_OPENSSL_BACKEND")] = self.options.get_safe("with_openssl", False)
+                self._cmake.definitions[self._get_cmake_option("CPR_FORCE_WINSSL_BACKEND")] = self.options.get_safe("with_winssl", False)
+
+            supports_any_ssl = self.options.get_safe("with_openssl", False) or self.options.get_safe("with_winssl", False)
+            if not self._uses_old_cmake_options and not supports_any_ssl:
+                self._cmake.definitions["CPR_ENABLE_SSL"] = False
             self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 
-    def build(self):
+    def validate(self):
+        if not self._uses_valid_abi_and_compiler:
+            raise ConanInvalidConfiguration("Cannot compile CPR/1.6.0 with libstdc++ on clang < 9")
+
+        if self.options.get_safe("with_openssl", False) and tools.is_apple_os(self.settings.os):
```

Comment:
It was fixed in master yesterday... 

```suggestion
        if self.options.get_safe("with_openssl", False) and tools.is_apple_os(self.settings.os) and tools.Version(self.version) == "1.6.0":
```

Reply:
Yep! Saw that PR, wasn't sure if I should come back or change it when 1.7.0 rolls out (which is what it's slated for). Happy to do this